car_steering_feedback_node:
  ros__parameters:
    can_interface: "can1"
    steering_conversion_factor: 15.7
